.ddev/config.selenium-standalone-chrome.yaml

```diff
@@ -0,0 +1,15 @@
+#ddev-generated
+# Remove the line above if you don't want this file to be overwritten when you run
```

Comment:
```suggestion
```

We should remove this, as I don't suppose we let ddev update this when there are updates.